tasks/1_golf-score/golf-score.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * GOLF_SCORES - enum, encounteres different nicknames
+ */
+export const GOLF_SCORES = {
+  HOLE_IN_ONE: 'Hole-in-one!',
+  EAGLE: 'Eagle',
+  BIRDIE: 'Birdie',
+  PAR: 'Par',
+  BOGEY: 'Bogey',
+  DOUBLE_BOGEY: 'Double Bogey',
+  GO_HOME: 'Go Home!',
+};
+
+/**
+ * golf-score - returns nickname, depending on
+ * how far above or below par your strokes are.
+ *
+ * @param {Number} par
+ * @param {Number} strokes
+ * @return {String}
+ */
+export function golfScore(par, strokes) {
+  if (strokes === 1) {
+    return GOLF_SCORES.HOLE_IN_ONE;
+  }
+  if (strokes <= par - 2) {
+    return GOLF_SCORES.EAGLE;
+  }
+  switch (strokes) {
```

Comment:
What is the reason of segregating some conditions in different `switch case` construction? At the same time some conditions placed in `if` statements

Reply:
You are right. The switch statement is redundant there. I fixed it.